maint-lib/makelib.mk

```diff
@@ -1,28 +1,32 @@
 # Container images built for each flavor
 IMAGES_TO_BUILD := daemon-base daemon
 
+HOST_ARCH ?= $(shell uname --machine)
```

Comment:
Why not naming it ARCH or rename ARCH to HOST_ARCH ? This double name for the same thing is confusing.

Do you envision that some override HOST_ARCH ?

Reply:
`HOST_ARCH` here can be overridden easily in a future where we are doing crossbuilds. E.g.,

```
make HOST_ARCH=x86_64 build
make HOST_ARCH=aarch64 build
```

As for the double naming, I would rather rename our current usage of `ARCH` to `HOST_ARCH`, as just `ARCH` could be vague.

Reply:
I'm fine with changing everything to HOST_ARCH but having two name of the same thing is disturbing.